### PR TITLE
Fix warnings in dev mode

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Fixed different React warnings we get while running in develop mode.

```
/eth/ethereum-org-website/src/components/ActionCard.js
  3:23  warning  'getImage' is defined but never used  no-unused-vars

/eth/ethereum-org-website/src/components/Morpher.js
  152:6  warning  React Hook useEffect has missing dependencies: 'morphTimeout', 'morpher', 'state.text', and 'state.words'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

/eth/ethereum-org-website/src/components/StablecoinAccordion.js
  144:7  warning  'InfoTitle' is assigned a value but never used  no-unused-vars

/eth/ethereum-org-website/src/components/StatsBoxGrid.js
  200:11  warning  No duplicate props allowed
                                                                                                      react/jsx-no-duplicate-props
  424:6   warning  React Hook useEffect has missing dependencies: 'ethPrices', 'formatNodes', 'formatPrice', 'formatTVL', 'formatTxs', 'nodes', 'txs', and 'valueLocked'. Either include them or remove the dependency array. You can also do
a functional update 'setEthPrices(e => ...)' if you only need 'ethPrices' in the 'setEthPrices' call  react-hooks/exhaustive-deps

/eth/ethereum-org-website/src/hooks/useActiveHash.js
  37:6  warning  React Hook useEffect has missing dependencies: 'itemIds' and 'rootMargin'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

/eth/ethereum-org-website/src/pages/community.js
   62:1  warning  In page templates only a default export of a valid React component and the named exports of a page query, getServerData or config are allowed.
        All other named exports will cause Fast Refresh to not preserve local component state and do a full refresh.

        Please move your other named exports to another file. Also make sure that you only export page queries that use the "graphql" tag from "gatsby".
  limited-exports-page-templates
  188:7  warning  'H3' is assigned a value but never used
                                                                                                                                                                                                    no-unused-vars

/eth/ethereum-org-website/src/pages/eth2/get-involved/index.js
  179:6  warning  React Hook useEffect has a missing dependency: 'consensusClients'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/eth/ethereum-org-website/src/pages/get-eth.js
  27:7  warning  'Title' is assigned a value but never used  no-unused-vars

/eth/ethereum-org-website/src/pages/stablecoins.js
  776:15  warning  Imported JSX component APY must be in PascalCase  react/jsx-pascal-case

/eth/ethereum-org-website/src/templates/static.js
  137:3  warning  Duplicate key 'Link'  no-dupe-keys

/eth/ethereum-org-website/src/templates/tutorial.js
  3:10  warning  'useIntl' is defined but never used  no-unused-vars

/eth/ethereum-org-website/src/templates/use-cases.js
    3:10  warning  'useIntl' is defined but never used               no-unused-vars
  102:7   warning  'LastUpdated' is assigned a value but never used  no-unused-vars

```

### CODE REVIEW

File: src/pages/stablecoins.js [hunk 1]
```diff
@@ -239,7 +239,7 @@ const ButtonColumn = styled.div`
   flex-direction: column;
 `
 
-const APY = styled.p`
+const Apy = styled.p`
```
Comment (minimalsm) [inline src/pages/stablecoins.js:242]: Bikeshedding but I'd prefer to just disable the rule here and use the acronym properly.

**Review by minimalsm — APPROVED** [review]
Lovely clean console 😬❤️

**Review by corwintines — APPROVED** [review]

**Comment by corwintines** [comment]
So fresh, so clean

**Comment by samajammin** [comment]
YAY thanks @pettinarip 🎉

**Pull request merged**